### PR TITLE
refactor: convert getConnection to non-hook

### PR DESCRIPTION
**Pull request by cartcrom**
## Description

Converts `useGetConnection` to a non-hook `getConnection` function

### CODE REVIEW

File: src/components/NavBar/ChainSelector.tsx [hunk 1]
```diff
@@ -53,9 +53,7 @@ function getChainsFromEIP155Accounts(accounts?: string[]): SupportedChainId[] {
 function useWalletSupportedChains() {
   const { connector } = useWeb3React()
 
-  const getConnection = useGetConnection()
-
-  const connectionType = getConnection(connector).type
+  const connectionType = getConnection(connector)?.type
```
Comment (zzmp) [inline src/components/NavBar/ChainSelector.tsx:56]: Is the optional operator necessary?

File: yarn.lock [hunk 2]
```diff
@@ -6157,28 +6157,6 @@
     "@walletconnect/types" "^1.8.0"
```
Comment (zzmp) [inline yarn.lock:6157]: This is fine, but should ideally be done in a separate PR.

**Review by zzmp — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (cartcrom) [reply yarn.lock:6157]: this wasn't intentional, I think it just happened from running yarn?

File: src/components/NavBar/ChainSelector.tsx (hunk 1 above)
Comment (cartcrom) [reply src/components/NavBar/ChainSelector.tsx]: You're right it's not necessary, I'm not sure why vscode was complaining about it originally

**Review by zzmp — APPROVED** [review]
